.gitlab/internal_image_deploy.yml

```diff
@@ -5,22 +5,25 @@
 docker_trigger_internal:
   stage: internal_image_deploy
   rules:
-    !reference [.on_deploy_stable_or_beta_repo_branch_a7]
+    !reference [.on_deploy_a7]
   needs:
     - job: docker_build_agent7_jmx
       artifacts: false
     - job: docker_build_agent7_jmx_arm64
       artifacts: false
-  trigger:
-    project: DataDog/images
-    branch: master
-    strategy: depend
-  variables:
-    IMAGE_VERSION: tmpl-v4
-    IMAGE_NAME: datadog-agent
-    RELEASE_TAG: ${CI_COMMIT_REF_SLUG}-jmx
-    BUILD_TAG: ${CI_COMMIT_REF_SLUG}-jmx
-    TMPL_AGENT_SRC_IMAGE: v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-jmx
-    TMPL_AGENT_SRC_REPO: ci/datadog-agent/agent-release
-    RELEASE_STAGING: "true"
-    RELEASE_PROD: "true"
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:main"]
+  script: # We can't use the 'trigger' keyword on manual jobs, otherwise they can't be run if the pipeline fails and is retried
+    - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_pipelines_scheduler_token --with-decryption --query "Parameter.Value" --out text)
+    - python3 -m pip install -r requirements.txt
+    - KUBERNETES_SERVICE_ACCOUNT_OVERWRITE="images"
```

Comment:
If you use `pipeline.trigger-child-pipeline` task, then I believe you won't need to overwrite `KUBERNETES_SERVICE_ACCOUNT_OVERWRITE`, since the only variables we pass to the child pipeline are variables defined in the `--variables` option (vs. using the `trigger` keyword, which passes all variables to the child pipeline). 

We can keep it here for now (it doesn't hurt), and look into removing it later.